backend/index.js

```diff
@@ -0,0 +1,126 @@
+import * as THREE from 'three'
+import * as Shaders from './shaders'
+import saveAs from 'file-saver'
+
+// controls: wasd + rf + qe + mouse
+require('three-fly-controls')(THREE)
+
+var container, camera, scene, renderer, controls
```

Comment:
nitpick: Use `let` instead of `var`

Reply:
made container local but the other ones are used in multiple functions so they were just capitalized to reflect that